server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -866,6 +866,13 @@ public static IndexMergePolicy fromString(String text) {
         Property.Dynamic
     );
 
+    public static final Setting<Boolean> PREFETCH_DOCS_DURING_FETCH_ENABLED = Setting.boolSetting(
```

Comment:
Since this file is annotated with public API, please label this as experimental so easy to update or remove in the future. Similar for other public methods and fields, and in SearchContext class also.